test/Bridges/lazy_bridge_optimizer.jl

```diff
@@ -993,6 +996,8 @@ Objective function of type `MOI.ScalarQuadraticFunction{$T}` is not supported an
               MOI.Utilities.replace_acronym(
             """
 Objective function of type `MOI.ScalarQuadraticFunction{$T}` is not supported and cannot be bridged into a supported objective function by adding only supported constrained variables and constraints. See details below:
+ [1] constrained variables in `MOI.Reals` are not supported because no added bridge supports bridging it.
+   Cannot add free variables and then constrain them because free variables are bridged but no functionize bridge was added.
```

Comment:
This is wrong, you cannot bridge free variables with the `Constraint.Functionize` bridge

Reply:
Hmm yeah. I haven't changed the error messages. This is just what was printed. I think I have a much better handle on the graph system now. It was a pretty easy mistake to omit these from being printed when we wrote this debugger!